src/services/anchor-service.ts

```diff
@@ -310,10 +318,12 @@ export default class AnchorService {
         request = requests[index];
 
         docId = this._getRequestDocID(request)
+        logger.debug(`About to load ceramic doc ${docId.baseID.toString()} at commit ${docId.commit.toString()}`)
         const doc = await this.ceramicService.loadDocument(docId)
         if (!doc) {
           throw new Error(`No valid ceramic document found with docId ${docId.toString()}`)
         }
+        logger.debug(`Successfully loaded ceramic doc ${docId.baseID.toString()} at commit ${docId.commit.toString()}`)
```

Comment:
Can we include some info about the doc? Maybe the size? Are there other things that are reasonable to log for this specific issue we are seeing now?

Reply:
hard to say.  I guess I could get an approximation of the doc size in there by using DoctypeUtils.serializeState() to get a string representation then taking the length of that string.  @oed any thoughts here?